absl/hash/internal/hash.h

```diff
@@ -564,6 +569,18 @@ H AbslHashValue(H hash_state, std::basic_string_view<Char> str) {
 
 #endif  // ABSL_HAVE_STD_STRING_VIEW
 
+#if defined(__cpp_lib_filesystem)
+
+template <typename H>
+H AbslHashValue(H hash_state, const std::filesystem::path& path) {
+  using PathView = std::basic_string_view<std::filesystem::path::value_type>;
+  PathView view = path.native();
+  return H::combine(
```

Comment:
It would be nice to simply delegate this call to `AbslHashValue(hash_state, view)` but delegation is explicitly forbidden in runtime.

Hence the copy-paste.